helper.js

```diff
@@ -0,0 +1,12 @@
+module.exports.returnOrThrow = function(getter, minimalModelerVersion) {
```

Comment:
Hmm I tried out having ESModules in the `vendor` exports but it doesn't really work unless we upfront specify what we want to export. So this simply does not work:

```javascript
import { returnOrThrow } from '../helper';

const exported = returnOrThrow(() => window.propertiesPanel.bpmn, '5.0.0');

export * from exported;
```

To make it work, we'd have to explicitly expose names of the exports, e.g.

```javascript
import { returnOrThrow } from '../helper';

const exported = returnOrThrow(() => window.propertiesPanel.bpmn, '5.0.0');

export const useService = exported.useService;
export const otherExport = exported.otherExport;
```

But then, this would mean that for each feature release of any of the vendor deps, we have to look into the exported components and update them in this repo.

Reply:
```
ERROR in ../camunda-modeler-plugin-helpers/vendor/bpmn-js-properties-panel.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: /Users/maciej/workspace/bpmn-io/camunda-modeler-plugin-helpers/vendor/bpmn-js-properties-panel.js: Unexpected token (5:14)

  3 | const result = returnOrThrow(() => window.propertiesPanel.bpmn, '5.0.0');
  4 |
> 5 | export * from result;
    |               ^
  6 |
```

Reply:
Let's ditch my idea for the moment in this case.